airlock/business_logic.py

```diff
@@ -41,6 +41,11 @@ class RequestFileType(Enum):
     SUPPORTING = "supporting"
 
 
+class FileApprovalStatus(Enum):
+    APPROVED = "APPROVED"
+    REJECTED = "REJECTED"
```

Comment:
I think we may also need an UNKNOWN status here.

When the review is created, it will be one of APPROVED or REJECTED.

But if the file is later updated, I think we want to *reset* the Review back to something neutral (we probably do not want to delete it).

Don't think we should implement that logic here, but wdyt about adding that status now?

Reply:
Hmmm, in the `local_db` implementation, the `FileReview` is attached to `RequestFileMetadata`, which includes the `file_id` which identifies a specific version of a file. So I /think/ if the file is later updated, a review will exist for an earlier version of the file but not for the new version of the file. It might be worth pushing this logic out of the DAL & into the BLL, but I don't think that's for this PR?

Reply:
oh right, yes, good point. yeah problem for later.

It does mean we will want to keep the old FileRequestMetadata around when a file is updated, perhaps.